src/server/utilities/cspHeader/index.js

```diff
@@ -416,8 +416,14 @@ const injectCspHeader = (req, res, next) => {
   const originHeader = req.headers['bbc-origin'];
   const { origin } = getOriginContext(originHeader);
 
-  const isLive =
-    origin === 'https://www.bbc.co.uk' || origin === 'https://www.bbc.com';
+  /**
+   * If we're handling requests externally, i.e. from Mozart, we can check the value
+   * of `originHeader` to determine whether we're in a Live environment. If this is
+   * not set we could be handling requests locally and should check SIMORGH_APP_ENV.
+   */
+  const isLive = originHeader
+    ? origin === 'https://www.bbc.co.uk' || origin === 'https://www.bbc.com'
```

Comment:
Thinking about it now........ why do we ever need to check the origin header? Can we not just rely on the SIMORGH_APP_ENV value?

Reply:
That's an interesting thought... 🤔
This would require an update to tests, but I don't see why we couldn't rely on `SIMORGH_APP_ENV`. It would resolve the disparity between origin and environment too.

Reply:
Im up for updating this then. Would be worth sharing with a group group of people to ensure we havent missed anything

Reply:
That's a thought. 🤔

It would require an update to tests, but I don't see why we couldn't rely on `SIMORGH_APP_ENV` either. It would help convey the differences between origin and environment too. I'll make some changes.

Reply:
Im up for updating this then. Would be worth sharing with a group group of people to ensure we havent missed anything